app/containers/BudgetGrid/index.js

```diff
@@ -19,6 +21,11 @@ export class BudgetGrid extends React.Component<BudgetGridProps> {
     categories: {},
   };
 
+  handleOnRowClick = (transactionId: number): void => {
+    const { history } = this.props;
+    history.push(`/budget/${transactionId}`);
```

Comment:
choosed **onClick** handler instead of **Link** since we cant wrap **tr** elements with **a** (seemed a cleaner solution instead of having Link in each table cell)